Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:18.2.0-alpine AS build
+FROM --platform=amd64 node:18.2.0-alpine AS build
```

Comment:
This is not a very portable way to solve the issue. I could imagine that CI/CD would build and publish the docker images on actual ARM machines at some point in the future instead of using buildx/qemu; but this is indeed sufficient for now.